journal-service/src/main/resources/application.yml

```diff
@@ -0,0 +1,6 @@
+spring:
+  application:
+    name: Kioke Journal Service
+
+server:
+  port: ${JOURNAL_SERVICE_PORT}
```

Comment:
These two lines can be removed since it's going to be added via environment variables.